File: system-test/dotnet/SystemTests/Legacy/Mod09/SmokeTests/External/ClockSmokeTest.cs
using SystemTests.Legacy.Mod09.Base;
using Xunit;

namespace SystemTests.Legacy.Mod09.SmokeTests.External;

public class ClockSmokeTest : BaseScenarioDslTest
{
    [Fact]
    public async Task ShouldBeAbleToGoToClock()
    {
        await Scenario().Assume().Clock().ShouldBeRunning();
    }
}

File: system-test/java/src/test/java/com/optivem/eshop/systemtest/latest/smoke/external/ClockSmokeTest.java
package com.optivem.eshop.systemtest.latest.smoke.external;

import com.optivem.eshop.systemtest.latest.base.BaseScenarioDslTest;
import org.junit.jupiter.api.Test;

class ClockSmokeTest extends BaseScenarioDslTest {
    @Test
    void shouldBeAbleToGoToClock() {
        scenario.assume().clock().shouldBeRunning();
    }
}


